src/ambient/ConnectableObservable.ts

```diff
@@ -0,0 +1 @@
+export interface ConnectableObservable<T> {}
```

Comment:
maybe this declarations could be single module such as

*ambient.ts*
```
export interface A{}
export interface B{}
export interface C{}
```
to allow single import to declarations like

`import {A,B,C} from 'ambient'`